frappe/public/js/frappe/views/reports/reportview.js

```diff
@@ -600,6 +601,45 @@ frappe.views.ReportView = frappe.ui.Listing.extend({
 			}, true);
 		}
 	},
+	
+	// save as
+	make_save_as: function() {
+		var me = this;
```

Comment:
I tried to merge the two save and save as, it was getting too conditional so copy pasted. :P
Should I try to merge it?